schemas/component.json

```diff
@@ -260,20 +260,66 @@
           "$ref": "common.json#/$defs/materialComposition",
           "description": "The chemical elements and basic materials which are part of the component."
         },
-        "definitionOfSurfacesAndPrimeDirection": {
-          "title": "Definition of surfaces and prime direction",
-          "description": "The prime surface and the prime direction are used to define the orientation of the component. If there is a `profileAngle` symmetry, the prime direction must be equal to the direction of the profile angle symmetry. The reference can also define which layers are switchable e.g. electrochromic.",
+        "prime": {
+          "title": "Prime surface and/or direction",
+          "description": "The prime surface and the prime direction are used to define the orientation of the component. Both may be defined with a reference and/or description. The two references may point to the same resource if it contains both the definition of prime surface and prime direction.",
+          "type": "object",
+          "properties": {
+            "surface": {
+              "title": "Prime surface",
+              "type": "object",
+              "properties": {
+                "reference": {
+                  "$ref": "common.json#/$defs/reference",
+                  "title": "Reference",
+                  "description": "Reference that defines which surface of the component is defined as non-prime and which as prime. Often, the non-prime surface faces the exterior and the prime surface faces the interior."
+                },
+                "description": {
+                  "type": "string",
+                  "title": "Description",
+                  "description": "Description of which sides of the components are defined as non-prime and which as prime. Often, the non-prime side faces the exterior and the prime side the interior."
+                }
+              },
+              "minProperties": 1,
+              "required": []
+            },
+            "direction": {
+              "title": "Prime direction",
+              "description": "The prime direction is used to define the orientation of the component. If there is a `profileAngle` symmetry, the prime direction must be equal to the direction of the profile angle symmetry.",
+              "type": "object",
+              "properties": {
+                "reference": {
+                  "$ref": "common.json#/$defs/reference",
+                  "title": "Reference",
+                  "description": "Reference that defines the prime direction that is used to calculate the azimuth of the direction of incidence."
+                },
+                "description": {
+                  "type": "string",
+                  "title": "Description",
+                  "description": "Description of which direction is defined as the prime direction."
```

Comment:
Please add the sentence "For a facade component, the prime direction is often defined as the vertical direction from the bottom to the top of the component."

Reply:
Added in https://github.com/building-envelope-data/api/pull/341/commits/4cb85d9f66f8b84c5de66b2895064463c2e35f8d